lua/nvim-web-devicons/icons-default.lua

```diff
@@ -3491,6 +3485,24 @@ local icons_by_operating_system = {
     cterm_color = "35",
     name = "Manjaro",
   },
+  ["mint"] = {
+    icon = "󰣭",
+    color = "#66af3d",
+    cterm_color = "70",
+    name = "Mint",
+  },
+  ["mojo"] = {
```

Comment:
Apologies, I should have noticed earlier:

This is in the `icons_by_operating_system` wheras it should be in `icons_by_file_extension`.

You can run `:NvimWebDeviconsHiTest` to check.

Reply:
Whoops. My fault. Just seen one big table and looked for the "m" section :see_no_evil:

Reply:
Thanks for the review work :100: !